packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.h

```diff
@@ -568,8 +569,9 @@ namespace Ioss {
       }
     }
 
-    virtual std::vector<size_t> get_all_block_field_data(const std::string &field_name, void *data,
-                                                         size_t data_size) const;
+    virtual std::vector<size_t> get_entity_field_data(const std::string &field_name,
+                                                      const std::vector<Ioss::ElementBlock*>& elem_blocks,
+                                                      void *data, size_t data_size) const;
```

Comment:
Did you plan to use `ElementBlockContainer` here?  I see the typedef at line 57.

Reply:
That typedef is probably no longer needed ... I can remove it if you want